packages/adapter-static/index.cjs

```diff
@@ -1,4 +1,4 @@
-module.exports = function ({ pages = 'build', assets = 'build' } = {}) {
+module.exports = function ({ pages = 'build', assets = pages, fallback = null } = {}) {
```

Comment:
Why did this file need to be renamed to `.cjs`?

Reply:
So it can be required by the config file https://github.com/sveltejs/kit/pull/1181/files#diff-ffa86c05d95319420b7940aecaf8b3f028dad6acf5e1ed7005b0955b01325801